code/game/atoms.dm

```diff
@@ -1906,3 +1906,6 @@
 		luminosity = max(max(base_luminosity, affecting_dynamic_lumi), 1)
 	else
 		luminosity = max(base_luminosity, affecting_dynamic_lumi)
+
+/atom/movable/proc/get_orbitable()
+	. = src
```

Comment:
```suggestion
	return src
```
Return is just much cleaner in these cases (and avoids an unnecessary assignment, not that that matters in a cold path like this)